polkadot/runtime/parachains/src/configuration.rs

```diff
@@ -682,18 +676,7 @@ pub mod pallet {
 			Self::set_coretime_cores_unchecked(new)
 		}
 
-		/// Set the max number of times a claim may timeout on a core before it is abandoned
-		#[pallet::call_index(7)]
-		#[pallet::weight((
-			T::WeightInfo::set_config_with_u32(),
-			DispatchClass::Operational,
-		))]
-		pub fn set_max_availability_timeouts(origin: OriginFor<T>, new: u32) -> DispatchResult {
-			ensure_root(origin)?;
-			Self::schedule_config_update(|config| {
-				config.scheduler_params.max_availability_timeouts = new;
-			})
-		}
+		// Call index 7 used to be `set_max_availability_timeouts`, which was removed.
```

Comment:
I believe this is fine because the other call indices remain the same.
I've seen that other PRs did this as well, like https://github.com/paritytech/polkadot-sdk/pull/5439/files